Mathlib/Order/WithBot.lean

```diff
@@ -249,16 +249,17 @@ section LE
 
 variable [LE α] {x y : WithBot α}
 
-/-- The order on `WithBot α`, defined by `⊥ ≤ ⊥`, `⊥ ≤ ↑a` and `a ≤ b → ↑a ≤ ↑b`.
-
-Equivalently, `x ≤ y` can be defined as `∀ a : α, x = ↑a → ∃ b : α, y = ↑b ∧ a ≤ b`,
-see `le_if_forall`. The definition as an inductive predicate is preferred since it
-cannot be accidentally unfolded too far. -/
+/-- Auxiliary definition for the order on `WithBot`. -/
```

Comment:
If this is auxiliary, why not just private it?

Reply:
No, not really. You're right, I forgot to mention that. I think it's good for it not to be private